lib/Db/Item.php

```diff
@@ -291,7 +291,7 @@ public function setBody(string $body = null)
     {
         // FIXME: this should not happen if the target="_blank" is already
         // on the link
-        $body = str_replace('<a', '<a target="_blank" rel="noreferrer"', $body);
+        $body = str_replace('<a ', '<a target="_blank" rel="noreferrer"', $body);
```

Comment:
```suggestion
        $body = str_replace('<a ', '<a target="_blank" rel="noreferrer" ', $body);
```

Since we are replacing an additional space here shouldn't we add that space to the replacement string too?